client/src/components/Dataset/DatasetName.test.js

```diff
@@ -1,35 +1,45 @@
 import { mount } from "@vue/test-utils";
 import DatasetName from "./DatasetName";
+import { getNewAttachNode } from "../../../tests/jest/helpers";
```

Comment:
import { getNewAttachNode } from "jest/helpers";
```suggestion
import { getNewAttachNode } from "jest/helpers";
```

Reply:
Good catch; I think I did this when moving stuff around and forgot to clean it up.